pyccel/parser/parser.py

```diff
@@ -89,28 +107,29 @@ def module_parser(self):
         return self._module_parser
 
     def parse(self, d_parsers=None, verbose=False):
```

Comment:
Can you please add a short docstring, complete of the "Parameters" and "Returns" sections?

Reply:
done